### PR TITLE
Improved billboards example, highlighting rotation and draw order

### PR DESCRIPTION
**Pull request by nobytesgiven**

### CODE REVIEW

File: examples/models/models_billboard.c [hunk 1]
```diff
@@ -59,8 +80,15 @@ int main(void)
 
                 DrawGrid(10, 1.0f);        // Draw a grid
 
-                DrawBillboard(camera, bill, billPosition, 2.0f, WHITE);
-
+                // Draw order matters!
+                if (distanceStatic > distanceRotating) {
```
Comment (raysan5) [inline examples/models/models_billboard.c:84]: Please, could you review [coding conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md), raylib uses aligned brackets.

Comment (nobytesgiven) [reply examples/models/models_billboard.c]: Sorry, I missed this one. It should follow the coding conventions now.

Comment (raysan5) [reply examples/models/models_billboard.c]: Excuse my insistence but same issu with the else, it should be
```c
if ()
{

}
else
{

}
```

Comment (nobytesgiven) [reply examples/models/models_billboard.c]: No, you're right. I don't usually write in this style and can't spot all of them. I should set up a linter. Sorry for wasting your time.

Comment (raysan5) [reply examples/models/models_billboard.c]: No worries! no time wasted! 👍🙂

**Comment by raysan5** [comment]
@nobytesgiven thanks for the improvement

**Pull request merged**